packages/antd/src/templates/ObjectFieldTemplate/index.js

```diff
@@ -62,12 +61,12 @@ const ObjectFieldTemplate = ({
         ? 24
         : 12;
 
-    if (_.isObject(colSpan)) {
+    if (colSpan instanceof Object) {
       return (
         colSpan[widget] || colSpan[field] || colSpan[type] || defaultColSpan
       );
     }
-    if (_.isNumber(colSpan)) {
+    if (typeof colSpan === "number") {
```

Comment:
I'm pretty sure this isn't a 1:1 change given that `isNumber` is implemented as:
```
function isNumber(value) {
  return typeof value == 'number' ||
    (isObjectLike(value) && baseGetTag(value) == numberTag);
}
```